index.html

```diff
@@ -6,7 +6,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <title>PyPy.js</title>
+  <title>PyPy.js - console</title>
```

Comment:
I'd prefer not to say "console" here since it's the top-level page of the site, just "PyPy.js" is probably the most appropriate title